hw7/ex34.py

```diff
@@ -0,0 +1,9 @@
+def fun(words):
+    return sum(1 for i in words if i in 'аеёиоуыэюя')
+song = "пара-ра-рам рам-пам-папам па-ра-па-да"
+s = song.lower().split()
+t = fun(s[0])
+if all([fun(i) == t for i in s]):
```

Comment:
Функция `all` умеет работать напрямую с генераторами.
```suggestion
if all(fun(i) == t for i in s):
```